pvlib/bifacial/infinite_sheds.py

```diff
@@ -0,0 +1,831 @@
+r"""
```

Comment:
This is a nice module docstring, but I don't think it's rendered anywhere. Did I miss it? I don't recall rendering any module docstrings (for better or worse). I do recall that it might not be super simple for our current documentation strategy. If so, perhaps the doc string could be copied into another place, like a new bifacial.rst or an example gallery py file?

Reply:
@cwhanse any comment on this? Seems like a good idea to me. I happy to move it if you agree. thx.

Reply:
+1 to bifacial.rst